docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,182 @@
+.. _metrics_guide:
+
+====================
+Metrics Guide
+====================
+
+This guide will help you understand how to control the metrics that validate and monitor your data with
+Deepchecks.
+The assessment of how well the model performs should drive informed decision making regarding the usage of that model,
+so we have to make sure that we measure the right things, i.e. the criteria that represnt tha key factors in the
+decision making process.
+
+**Structure:**
+
+* `How can I use metrics with Deepchecks? <#how-can-i-use-metrics-with-deepchecks>`__
+* `Default metrics <#default-metrics>`__
+* `Alternative metrics <#alternative-metrics>`__
+* `List of Supported Strings <#list-of-supported-strings>`__
+* `Custom Metrics <#custom-metrics>`__
+
+
+How can I use metrics with Deepchecks?
+======================================
+The metrics used in deepchecks are in accordance with sklearn scorer API for tabular checks and with Torch Ignite metric
+API for vision checks.
+To use metrics with Deepchecks suites and checks you can use the defaults, that fits the most common usecase, or pass
+alternative scorers that are more accurate for your specific usecase.
+
+Default metrics
+===============
+All of our relevant checks come with default metrics.
+
+The default scorers by task type are:
+
+Tabular:
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
+
+*   F1
+*   Precision
+*   Recall
+
+Regression:
+
+*   Negative RMSE (negative so larger is better)
+*   Negative MAE
+*   R2
+
+Vision:
+
+Classification:
+
+*   Precision
+*   Recall
+
+Object detection:
+
+*   Mean average precision
+*   Mean average recall
+
+To run a check with the default metrics, run it without passing the scorers.
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults aren’t good enough to describe the specifics of the problem.
+If this is the case, you can pass a list of metrics or a dict in the format {metric name: metric} as a parameter to the
```

Comment:
@noamzbr it's supposed to work that way but currently not all checks support strings, it will be correct https://github.com/deepchecks/deepchecks/issues/1743

Reply:
So I guess best we write it now as it's supposed to be